test/AspNetDotEnvConfigurationProvider.Test/AspNetDotEnvConfigurationProviderTest.cs

```diff
@@ -8,55 +11,57 @@ public class AspNetDotEnvConfigurationProviderTest
 		[Fact]
 		public void LoadKeyValuePairsFromFile()
 		{
-			//var dict = new Hashtable()
-			//	{
-			//		{"DefaultConnection:ConnectionString", "TestConnectionString"},
-			//		{"DefaultConnection:Provider", "SqlClient"},
-			//		{"Inventory:ConnectionString", "AnotherTestConnectionString"},
-			//		{"Inventory:Provider", "MySql"}
-			//	};
-			//var envConfigSrc = new AspNetDotEnvConfigurationProvider();
-
-			//envConfigSrc.Load(new FileInfo());
-
-			//Assert.Equal("TestConnectionString", envConfigSrc.Get("defaultconnection:ConnectionString"));
-			//Assert.Equal("SqlClient", envConfigSrc.Get("DEFAULTCONNECTION:PROVIDER"));
-			//Assert.Equal("AnotherTestConnectionString", envConfigSrc.Get("Inventory:CONNECTIONSTRING"));
-			//Assert.Equal("MySql", envConfigSrc.Get("Inventory:Provider"));
+			
+			var dict = new Hashtable()
+				{
+					{"DefaultConnection:ConnectionString", "TestConnectionString"},
+					{"DefaultConnection:Provider", "SqlClient"},
+					{"Inventory:ConnectionString", "AnotherTestConnectionString"},
+					{"Inventory:Provider", "MySql"}
+				};
+			
+			var envConfigSrc = new AspNetDotEnvConfigurationProvider("test.env");
+
+			envConfigSrc.Load();
+			
+			Assert.Equal("TestConnectionString", envConfigSrc.Get("defaultconnection:ConnectionString"));
+			Assert.Equal("SqlClient", envConfigSrc.Get("DEFAULTCONNECTION:PROVIDER"));
+			Assert.Equal("AnotherTestConnectionString", envConfigSrc.Get("Inventory:CONNECTIONSTRING"));
+			Assert.Equal("MySql", envConfigSrc.Get("Inventory:Provider"));
 		}
 
 
 		[Fact]
 		public void LastVariableAddedWhenMultipleEnvironmentVariablesWithSameNameButDifferentCaseExist()
 		{
-			//var dict = new Hashtable()
-			//	{
-			//		{"CommonEnv", "CommonEnvValue1"},
-			//		{"commonenv", "commonenvValue2"},
-			//		{"cOMMonEnv", "commonenvValue3"},
-			//	};
-			//var envConfigSrc = new EnvironmentVariablesConfigurationProvider();
-
-			//envConfigSrc.Load(dict);
-
-			//Assert.True(!string.IsNullOrEmpty(envConfigSrc.Get("cOMMonEnv")));
-			//Assert.True(!string.IsNullOrEmpty(envConfigSrc.Get("CommonEnv")));
+			var dict = new Hashtable()
+				{
+					{"CommonEnv", "CommonEnvValue1"},
+					{"commonenv", "commonenvValue2"},
+					{"cOMMonEnv", "commonenvValue3"},
+				};
+			var envConfigSrc = new AspNetDotEnvConfigurationProvider("test.env", true);
```

Comment:
Write dict hashtable to a temporary file  , e.g
```
var tempEnvFile = Path.GetTemporaryFilename();
using(var writer = File.OpenWrite(tempFile))
foreach(var kv in dict)
{
writer.WriteLine($"{kv.Key}={kv.Value}
}

```

Above can be put into a method and reused across tests